src/main/java/eu/siacs/conversations/ui/ConversationsOverviewFragment.java

```diff
@@ -370,4 +381,20 @@ private void setScrollPosition(ScrollState scrollPosition) {
 			layoutManager.scrollToPositionWithOffset(scrollPosition.position, scrollPosition.offset);
 		}
 	}
+
+	public List<Conversation> getConversations() {
+		return conversations;
+	}
+
+    public void changeBackgroundColor(boolean isSelected){
+	    for(int position=0; position < conversations.size(); position++){
+            ConversationAdapter.ConversationViewHolder vh = (ConversationAdapter.ConversationViewHolder) this.binding.list.findViewHolderForLayoutPosition(position);
+            if(isSelected){
+                vh.binding.frame.setBackgroundColor(Color.parseColor("#dfdfdf"));
```

Comment:
As I mentioned at ConversationsActivity:751, this color can be defined as constant